branchingQuestion.js

```diff
@@ -43,12 +43,66 @@ H5P.BranchingQuestion = (function () {
       var icon = document.createElement('img');
       icon.classList.add('h5p-branching-question-icon');
       icon.src = self.getLibraryFilePath('branching-question-icon.svg');
+      const close = createCloseElement();
 
       wrapper.appendChild(icon);
+      wrapper.appendChild(close);
 
       return wrapper;
     };
 
+    /**
+     * Create close element and register events
+     *
+     * @return {Element} close
+     */
+    const createCloseElement = function () {
+      const close = document.createElement('div');
```

Comment:
I believe we should try to use <button> instead. then we don't need to add special handling of the key events.